### PR TITLE
add: i18n and init. config i18n.js

### PR DESCRIPTION
**Pull request by HuziMuzi**
install  i18n, react-i18n and i18next-browser-languagedetector, add initial config i18n.js

### CODE REVIEW

File: __tests__/App-test.tsx [hunk 1]
```diff
@@ -16,11 +15,3 @@ jest.mock('react-native-splash-screen', () => {
     show: jest.fn(),
   };
 });
-
-describe(App, () => {
```
Comment (antoniosk10) [inline __tests__/App-test.tsx:20]: Why did you delete this test?

**Review by antoniosk10 — DISMISSED** [review]

**Review by KhaninPasha0488 — DISMISSED** [review]

**Review by Julia-Kovalchuk — DISMISSED** [review]

**Review by antoniosk10 — APPROVED** [review]

**Review by KhaninPasha0488 — APPROVED** [review]

**Review by Julia-Kovalchuk — APPROVED** [review]
